[WasmGC article] Add Flutter demo (#7704)
GoogleChrome/developer.chrome.com: site/en/blog/wasmgc/index.md

## Seeing a WasmGC-ported programming language in action

> ### Kotlin/Wasm

Reviewer: ```suggestion
### Kotlin and Wasm
```

> ### Kotlin and Wasm

Reviewer: ```suggestion
### Kotlin Wasm
```

Reviewer: Changed to match https://kotlinlang.org/docs/wasm-overview.html

In this early experimental stage, the [Kotlin/Wasm image viewer](https://kotlin-wasm-image-viewer.glitch.me/) demo is fully rendered onto a `canvas` with no meaningful DOM tree that could be used to create an [accessibility tree](https://developer.mozilla.org/docs/Glossary/Accessibility_tree). This means it's completely inaccessible to non-sighted users and breaks important browser features like find on page, translation, text selection, extensions, zooming, and link to text fragment.
{% endAside %}

> The Dart and Flutter teams at Google are also preparing support for WasmGC support. The Dart-to-Wasm compilation work is almost complete and the team is working on tooling support for delivering Flutter web applications compiled to WebAssembly. You can read about the current state of the work in the [Flutter documentation](https://flutter.dev/wasm). A demo showcasing the Flutter WasmGC Preview is embedded below.

Author: ```suggestion
The Dart and Flutter teams at Google are also preparing support for WasmGC. The Dart-to-Wasm compilation work is almost complete, and the team is working on tooling support for delivering Flutter web applications compiled to WebAssembly. You can read about the current state of the work in the [Flutter documentation](https://flutter.dev/wasm). A demo showcasing the Flutter WasmGC Preview is embedded below.
```

In this early experimental stage, the [Kotlin/Wasm image viewer](https://kotlin-wasm-image-viewer.glitch.me/) demo is fully rendered onto a `canvas` with no meaningful DOM tree that could be used to create an [accessibility tree](https://developer.mozilla.org/docs/Glossary/Accessibility_tree). This means it's completely inaccessible to non-sighted users and breaks important browser features like find on page, translation, text selection, extensions, zooming, and link to text fragment.
{% endAside %}

### Dart and Flutter

> The Dart and Flutter teams at Google are also preparing support for WasmGC. The Dart-to-Wasm compilation work is almost complete, and the team is working on tooling support for delivering Flutter web applications compiled to WebAssembly. You can read about the current state of the work in the [Flutter documentation](https://flutter.dev/wasm). The [Flutter WasmGC Preview](https://flutterweb-wasm.web.app/) demo is embedded below.

Reviewer: ```suggestion
The Dart and Flutter teams at Google are also preparing support for WasmGC. The Dart-to-Wasm compilation work is almost complete, and the team is working on tooling support for delivering Flutter web applications compiled to WebAssembly. You can read about the current state of the work in the [Flutter documentation](https://flutter.dev/wasm). The following demo is the [Flutter WasmGC Preview](https://flutterweb-wasm.web.app/).
```
